templates/homepage-events-band.tmpl

```diff
@@ -0,0 +1,33 @@
+<div class="grid-wrapper homepage-events-band">
+  <div class="width-12-12 width-12-12-m">
+    <div class="section-title">
+      <div class="section-top">
+        <img class="section-icon" src="/assets/images/calendar.svg" />
+        <h2>{{ homepage.events.title }}</h2>
+        <div class="section-view-more">
+          <a href="{{ homepage.events.url }}">{{ homepage.events.view_calendar }}&nbsp;<i class="fa fa-calendar-check" aria-hidden="true"></i></a>
```

Comment:
Really like the fa-calendar here, we should do this on all the sections...

Reply:
Addressing this with https://github.com/ansible-community/community-website/pull/100